### PR TITLE
Use GitHub actions

### PR DESCRIPTION
**Pull request by phadej**

### CODE REVIEW

File: distributive.cabal [hunk 1]
```diff
@@ -23,8 +23,8 @@ tested-with:   GHC == 7.0.4
              , GHC == 8.2.2
              , GHC == 8.4.4
              , GHC == 8.6.5
-             , GHC == 8.8.3
-             , GHC == 8.10.1
+             , GHC == 8.8.4
+             , GHC == 8.10.3
 extra-source-files:
   .hlint.yaml
   .travis.yml
```
Comment (RyanGlScott) [inline distributive.cabal:30]: Delete this line, as well as the `travis-cabal-apt-install` line below.

File: cabal.haskell-ci [hunk 2]
```diff
@@ -2,3 +2,13 @@ no-tests-no-benchmarks: False
 unconstrained:          False
 allow-failures:         <7.3
 irc-channels:           irc.freenode.org#haskell-lens
```
Comment (RyanGlScott) [inline cabal.haskell-ci:4]: Don't forget `irc-if-in-origin-repo: True`.

File: distributive.cabal (hunk 1 above)
Comment (phadej) [reply distributive.cabal]: Good catch. Done.

File: cabal.haskell-ci (hunk 2 above)
Comment (phadej) [reply cabal.haskell-ci:4]: done

**Comment by phadej** [comment]
I forgot that `distributive` is in the very bottom of dep-tree, and also not simple, (and maybe not safe).

**Pull request merged**